cypress/fixtures/addresses.json

```diff
@@ -37,6 +38,7 @@
     "postalCode": "37745",
     "country": "US",
     "countryArea": "TN",
+    "countryAreaFullName": "Tennessee",
```

Comment:
Why adding new filed "countryAreaFullName", not just change "countryArea" value from "TN", to "Tennessee"? In these way, we don't have to change anything in fillUpBasicAddress and in expectCorrectBasicAddress commands

Reply:
Right now country area is a select type so I need to make changes in fillUpBasicAddress and use fillAutocompleteSelect there.

I was thinking about only changing the country area from code like "TN" to full name "Tennessee" but when I ran tests I noticed that it fails on the assertion for the country area - the backend returns the country area as code (This is not a bug, it's just data normalization on BE)
<img width="1789" alt="image" src="https://user-images.githubusercontent.com/30683248/209553544-394bbbf7-6983-41ed-97ad-b236572b1190.png">

Reply:
In customer it work in other way. The backend returns the country area as full name. So tests for adding address in customer are not working after this changes. I think API should return the same value type in all places
![Uploading tN.png…]()

Reply:
In customer it work in other way. The backend returns the country area as full name. So tests for adding address in customer are not working after this changes. I think API should return the same value type in all places
![Uploading tN.png…]()

Reply:
You're right, I asked the backend team yesterday about this case and they confirmed that they should return the code here as in other mutations with the address: https://github.com/saleor/saleor/issues/11572